src/main/java/io/cdap/plugin/salesforce/SalesforceSchemaUtil.java

```diff
@@ -241,9 +245,23 @@ public static Schema getSchemaWithFields(SObjectDescriptor sObjectDescriptor,
     return Schema.recordOf("output", schemaFields);
   }
 
-  private static Schema getCdapFieldSchema(Field field) {
+  private static Schema createFieldSchema(Field field) {
     Schema fieldSchema = SALESFORCE_TYPE_TO_CDAP_SCHEMA.getOrDefault(field.getType(), DEFAULT_SCHEMA);
     return field.isNillable() ? Schema.nullableOf(fieldSchema) : fieldSchema;
   }
 
+  /**
+   * Normalize the given field name to be compatible with
+   * <a href="https://avro.apache.org/docs/current/spec.html#names">Avro names</a>
+   */
+  private static String normalizeAvroName(String name) {
```

Comment:
nit: this method should have tests.

Reply:
A avro parsing has been added to an existing test.